src/sentry/discover/models.py

```diff
@@ -73,3 +74,18 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_discoverkeytransaction"
         unique_together = (("project", "owner", "transaction"),)
+
+
+class TeamKeyTransaction(Model):
+    __core__ = False
```

Comment:
This will exclude key transactions from metadata/configuration backups made on self-hosted. Is that what you want?

Reply:
What's the criteria for evaluating if this data should be included in the backups? Is it just like 'hmm this seems important, so I shouldn't set it to False'

Reply:
I initially thought about making this `True`. However I noticed that all discover-related models have this as `False` and thought they should be consistent.